<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Align with target</string>
  </property>
  <layout class="QGridLayout" name="gridLayoutDialog">
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <property name="spacing">
    <number>16</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayoutHorizontal">
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="labelHorizontal">
       <property name="text">
        <string>Horizontal</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="hRightLeft">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/align-right-left.svg</normaloff>icons/align-right-left.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">horizontal</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="hRightCenter">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/align-right-center.svg</normaloff>icons/align-right-center.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">horizontal</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="hRightRight">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/align-right-right.svg</normaloff>icons/align-right-right.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">horizontal</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QToolButton" name="hCenterLeft">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/align-center-left.svg</normaloff>icons/align-center-left.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">horizontal</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="hCenterCenter">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/align-center-center.svg</normaloff>icons/align-center-center.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">horizontal</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="hCenterRight">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/align-center-right.svg</normaloff>icons/align-center-right.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">horizontal</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QToolButton" name="hLeftLeft">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/align-left-left.svg</normaloff>icons/align-left-left.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">horizontal</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QToolButton" name="hLeftCenter">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/align-left-center.svg</normaloff>icons/align-left-center.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">horizontal</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QToolButton" name="hLeftRight">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/align-left-right.svg</normaloff>icons/align-left-right.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">horizontal</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <layout class="QGridLayout" name="gridLayoutVertical">
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="labelVertical">
       <property name="text">
        <string>Vertical</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="vBottomTop">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/align-bottom-top.svg</normaloff>icons/align-bottom-top.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">vertical</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="vCenterTop">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/align-center-top.svg</normaloff>icons/align-center-top.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">vertical</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="vTopTop">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/align-top-top.svg</normaloff>icons/align-top-top.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">vertical</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QToolButton" name="vBottomCenter">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/align-bottom-center.svg</normaloff>icons/align-bottom-center.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">vertical</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="vCenterCenter">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/align-center-center.svg</normaloff>icons/align-center-center.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">vertical</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="vTopCenter">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/align-top-center.svg</normaloff>icons/align-top-center.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">vertical</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QToolButton" name="vBottomBottom">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/align-bottom-bottom.svg</normaloff>icons/align-bottom-bottom.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">vertical</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QToolButton" name="vCenterBottom">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/align-center-bottom.svg</normaloff>icons/align-center-bottom.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">vertical</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QToolButton" name="vTopBottom">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/align-top-bottom.svg</normaloff>icons/align-top-bottom.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">vertical</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="horizontal"/>
  <buttongroup name="vertical"/>
 </buttongroups>
</ui>
